client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -915,11 +902,10 @@ export default {
       }
     },
 
-    showHintAndDoExport (route) {
+    showHintAndDoExport ({ route, docxHeaders }) {
       if (window.dpconfirm(Translator.trans('export.statements.hint'))) {
-        window.location.href = this.exportRoute(route)
+        window.location.href = this.exportRoute(route, docxHeaders)
```

Comment:
could that be archived by an APi-request?

Reply:
it could be, but it should be adjusted in the BE,  it was implemented in a different way. Should it be done via API?

Reply:
not necessarily right away, but it would be better I guess. with that type of requests, we always got a page reload wich is not so user friendly

Reply:
hm, there is no page reload after exporting the files.

I do not know how complex it would be for the BE to switch to an Api, but it should have been done a bit earlier.. it could be discussed with the be, but I am not sure if it is necessary because there is no difference for the user. I do not see a page reload